osh/word_eval.py

```diff
@@ -887,19 +887,22 @@ def _EvalVarRef(self, val, blame_tok, quoted, vsub_state, vtest_place):
 
             elif case(value_e.Str):
                 val = cast(value.Str, UP_val)
-                bvs_part = self.unsafe_arith.ParseVarRef(val.s, blame_tok)
-                return self._VarRefValue(bvs_part, quoted, vsub_state,
-                                         vtest_place)
+                var = val.s
```

Comment:
How about calling this `bvs_str` or something?  I think that is what it is, it is a string that follows the syntax of what's inside `${...}`, the BracedVarSub

Reply:
Hmm, it needs to be a variable name of the form `name` or an array element `name[...]`. It cannot be like `a:-xx`, `a:offset:count`, or `a%.*`. How about `var_name_str` or `var_ref_str`?

Reply:
var_ref_str sounds good, thanks!

Reply:
Thanks! I updated the variable name: [`8cfe898..6ff4b1b`](https://github.com/oils-for-unix/oils/compare/8cfe8981cae3f6d2e67edce2245686a429e2d9f0..6ff4b1b22ace07b659bb9d98499836e5b07b0ef0).